<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tcp_server_dialog</class>
 <widget class="QDialog" name="Tcp_server_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="start_button">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="stop_button">
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="port_spin_box">
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>63999</number>
     </property>
     <property name="value">
      <number>2000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
